.azure-pipelines/client.yml

```diff
@@ -1,9 +1,19 @@
 trigger:
 - master
 
-# variables:
+variables:
 # ProjectFile - The project to build and test. This variable is defined in pipeline web ui because we want to be able to provide it at queue time and that isn't supported in yaml yet.
 
+ # public
+ - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+   - name: AzConfigConnectionString
+     value: ''
+ # internal
+ - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+   - group: AzureSDK KeyVault - AzConfig LiveTest
```

Comment:
As part of our [guidelines](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki?wikiVersion=GBwikiMaster&pagePath=%2FAzure%20DevOps%20Pipeline%20Guidance&pageId=2&anchor=variable-groups) we shouldn't include the group directly in the yaml file we should instead just link it in the build pipeline.

Reply:
Currently the pipeline is not setup for this. I'll leave this as-is for now and file an issue to conform to the guidelines -- would that work?

Reply:
I would just remove this as when you setup the pipeline you have the link the variable group regardless to make it work.

Reply:
Are you saying to remove the line with the group?

Reply:
Exactly all you should need to do is delete the one line with `-group ...`.